crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -90,6 +90,23 @@ def addons
       end
     end
 
+    def repocheck(options = {})
+      platform = options.fetch(:platform)
+      arch = options.fetch(:arch)
+
+      repos = {}
+      wanted_repos = ["cloud", "sles12-sp2-pool", "sles12-sp2-updates"]
```

Comment:
Do we need a fixed list here, or could we obtain it from a list of repositories currently available (via a repo library)? (We would need to change sp1 to sp2 obviously)